docs/source/index.rst

```diff
@@ -94,7 +94,7 @@ DAML SDK documentation
    :caption: Experimental features
 
    experimental/warning
-   daml-integration-kit/index
+   tools/ledger-api-test-tool/index
```

Comment:
You are adding a new docs page, but don't mention it in the release notes. Are you intentionally silently changing integration kit to api-test-tool?

Reply:
the ledger-api-test-tool was previously a sub-link of the daml integration kit. the test tool itself works fine and is supported by us. therefore I'm fine that it is still listed in the docs.